wasm_runtime/src/config.rs

```diff
@@ -23,6 +23,7 @@ pub struct WasmConfig {
     pub wasi_envs:    Vec<(String, String)>,
     pub wasi_dirs:    Vec<String>,
     pub wasi_mapdirs: Vec<(String, String)>,
+    pub entrypoint:     String
```

Comment:
I'd move this property before all the WASI ones, since it belongs to the Wasm module itself.